GameServer/packets/Server/GSUDPPacketOut.cs

```diff
@@ -54,12 +54,6 @@ public override ushort WritePacketLength()
 		{
 			Position = 0;
 			WriteShort((ushort)(Length-5));
-
-			//IMPORTANT!!!
-			//Set the capacity of the internal buffer or
-			//the byte-array of GetBuffer will be TOO big!
-			Capacity = (int)Length;

```

Comment:
From the comments this looks important. Shouldn't we keep this? Or what is the effect of removing it?

Reply:
No, it's not needed anymore since I use the length from the packet's data so it save a buffer reallocation.